mmdet3d/models/dense_heads/vote_head.py

```diff
@@ -526,10 +554,11 @@ def get_targets_single(self,
         size_res_targets /= pos_mean_sizes
 
         mask_targets = gt_labels_3d[assignment]
+        center_all_targets = center_targets[assignment]
```

Comment:
Should we also use assigned_center_targets here for consistency?

Reply:
yes, my mistake